examples/Client/PublishSubscribe/PublishEventExample/PublishEventExample.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netcoreapp7.0</TargetFramework>
```

Comment:
Added this to make this testable. I can revert if maintainer things this is not a good change.